src/test/cli.ts

```diff
@@ -131,7 +132,15 @@ async function runTests(args: string[], opts: { [key: string]: any }) {
     runner.loadEmptyConfig(process.cwd());
   }
 
-  const result = await runner.run(!!opts.list, args.map(forceRegExp), opts.project || undefined);
+  const filePatternFilters: FilePatternFilter[] = args.map((arg) => {
+    const splitted = (typeof arg === 'string' && arg.includes(':')) ? arg.split(':') : null;
```

Comment:
Since we support regex here, I wonder how something like `/[ab:]/` is going to work? Perhaps, we should treat the string as a regexp, but match `file:line:column` instead of matching just `file`?

Reply:
for now I didn't add support for handling regular expressions. I think thats not needed there, since regexp would mean run it on multiple files and files usually are differently structured.